features/click-to-load.json

```diff
@@ -0,0 +1,18 @@
+{
+    "_meta": {
+        "description": "Click to Load feature for blocking embedded third-party social/video content.",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "site breakage"
+        }
+    },
+    "exceptions": [],
+    "settings": {
+        "Facebook, Inc.": {
+            "ruleActions": ["block-ctl-fb"]
```

Comment:
we don't need `clicksBeforeSimpleVersion` setting, from the click-to-play config, anymore?

Reply:
Nope, that's deprecated.